packages/storybook/src/nimble/select/select.stories.ts

```diff
@@ -154,6 +157,11 @@ const metadata: Meta<SelectArgs> = {
         </${selectTag}>
     `),
     argTypes: {
+        label: {
+            name: 'default',
+            description: `${slottedLabelDescription({ componentName: 'select' })}`,
```

Comment:
Reminder to work with @fredvisser to figure out usage guidance on when clients should use label vs placeholder vs neither. Doesn't have to block this PR but it'd be good to have a decision before closing the feature.